ruby-challenges.rb

```diff
@@ -19,12 +29,41 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+def sum_arrays(array_of_numbers)
+    array_of_numbers.sum
+end
+
+p sum_arrays(nums_array1)
+p sum_arrays(nums_array2)
+
 
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
 
+class Bike
+    attr_accessor :model, :wheels, :current_speed
+    def initialize(model , wheels = 2, current_speed = 0)
+        @model = model
+        @wheels = wheels
+        @current_speed = current_speed
+    end
+    def display_information
+        p "The #{@model} bike has #{@wheels} wheels and is going #{@current_speed} mph."
+    end
+    def pedal_faster(increase_rate)
+        @current_speed = @current_speed + increase_rate
+        p @current_speed
+    end
+    def brake(decrease_rate)
+        @current_speed = [@current_speed - decrease_rate, 0].max
+        p @current_speed
+    end
+end
+
```

Comment:
It is a best practice to always have the `p` on the method call rather than the inner logic.